include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -922,17 +923,6 @@ Operator<dim, Number>::solve_nonlinear(VectorType &       sol,
   // solve nonlinear problem
   auto const iter = newton_solver->solve(sol, update);
 
-  // TODO
```

Comment:
I removed this completely as it is commented, we looked at the constraints once again, and we tested matrix-free and matrix-based variants. All works.